backend/main.py

```
@@ -1,19 +1,20 @@
 from sqlalchemy.orm import Session
 from database import SessionLocal, engine
 from models import User
-from schemas import UserCreate, Token, LoginRequest, UserUpdate
+from schemas import UserCreate, Token, LoginRequest, UserUpdate, OrderOut, OrderCreate
 from utils import auth_utils, create_access_token, verify_token, create_email_verification_token, send_verification_email
 from datetime import timedelta
 import os
 import dotenv
 from pydantic import BaseModel
 import uvicorn
 from typing import Annotated, List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)